### DISCUSSION TITLE
How to find approval messages through proposal messages

### DISCUSSION THREAD

**Discussion by AoiIsMine** [filecoin-project/lotus#8297] [General Q&A]
I have the ID of the proposal message. When it is approved, I don't know which approval message corresponds to the proposal message and which interface or method I should use to find the association

Thanks for your help

**Comment by Q-Action** [comment]
TxID
<img width="297" alt="image" src="https://user-images.githubusercontent.com/17774429/158142215-f94a1073-3870-4f94-aa9c-d05980b3308a.png">

**Reply by AoiIsMine** [reply]
谢谢你的回复
就是找不到能返回审批消息的txId的接口来着
(lotus-shed msg倒是可以,但是好像没有对应json-rpc接口)

**Comment by TippyFlitsUK** [comment]
Hi @AoiIsMine 

Have you been able to resolve this?

Many thanks!

**Reply by AoiIsMine** [reply]
no~yet
maybe use other methods to judge whether the proposal message corresponds to the approval message, but i don't know whether there is a directly available interface

**Reply by TippyFlitsUK** [reply]
Many thanks for your response @AoiIsMine!
